ConsoleApp1/ConsoleApp1/bin/Debug/netcoreapp3.1/ConsoleApp1.deps.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Add .gitignore file to not push such changes.

Reply:
added to order. hope it works this way